app-admin/prelude-manager/metadata.xml

```diff
@@ -24,5 +24,7 @@
 		<flag name="dbx">Enables Prelude Manager storing Prelude
 		alerts into database through <pkg>dev-libs/libpreludedb</pkg>
 		.</flag>
+        <flag name="libev">Support event loops using <pkg>dev-libs/libev</pkg>
```

Comment:
Per https://github.com/Prelude-SIEM/prelude-manager/blob/master/configure.ac#L242 it doesn't look like libev is optional. Am I interpreting it wrong? If that's the case, this description is confusing.

If `libev` is not optional, let's not provide support for embedded libev and unconditionally use the system one.